DirectoryListing/Directory/IndividualDetailController.swift

```diff
@@ -22,11 +22,19 @@ class IndividualDetailController: UIViewController {
 
         // Do any additional setup after loading the view.
         
-        assert(individual != nil)
+        guard let tempIndividual = individual else {
+            DLog("Individual must be set for individual detail controller.")
+            return
+        }
+        
+        guard let tempFullNameLabel = fullNameLabel, let tempBirthDateLabel = birthDateLabel, let tempAffiliationLabel = affiliationLabel else {
```

Comment:
You actually don't need to guard on these. You can just use a question mark instead of an exclamation point: `fullNameLabel?.text = ...`